stable/redis-ha/values.yaml

```diff
@@ -22,7 +22,7 @@ nodeSelector: {}
 tolerations: []
 
 ## Redis image version
-redis_image: quay.io/smile/redis:4.0.6r2
+redis_image: diegocn/redis-auth
```

Comment:
Could you contribute this change upstream to https://github.com/smileisak/docker-images/blob/master/redis/alpine/Dockerfile?

Reply:
PR opened on https://github.com/smileisak/docker-images/pull/12

Reply:
@diegocn PR smileisak/docker-images#12 merged

Reply:
Thanks @smileisak, mind tagging a release for that too?

Reply:
The latest tag (quay.io/smile/redis:latest) already contains the auth changes and redis version 4.0.8-r0. Can the latest be used in the chart?

Reply:
@diegocn we'd prefer to have a tagged version especially since this was using a tagged version before.